### DISCUSSION TITLE
[Request] Seafile

### DISCUSSION THREAD

**Discussion by Dvalin21** [tteck/Proxmox#1726] [Q&A]
Requesting for Seafile to be added.

It can be installed onto linux directly

https://manual.seafile.com/deploy/

Thanks

**Comment by danofun** [comment]
A Seafile LXC gets my vote as well. Great work being done here. Cheers!

**Reply by leonelafonsoe** [reply]
whats the difference with syncthing or nextcloud?

**Reply by danofun** [reply]
Syncthing is feature limited file synchronization solution between machines without utilizing a centralized, host server. Nextcloud has grown to become a hub for a multitude of seft-hosted solutions. Seafile is a server-centric file host with a feature set that meets advanced file sharing needs without trying to solve every self hosted need. I use Syncthing for a couple of folders I want to keep up-to-date between a few VM's and migrated from Nextcloud to Seafile for any collaborative files as well as my personal Docs. I'm very pleased with Seafile and think it would make a great addition to this community.

**Reply by buczi11** [reply]
Hi,
Personally, I have been thinking about these products for a longer time: owncloud, nextcloud, seafile and filecloud. 

Pros and cons for Seafile Server
+ For a central hosting system for multiple vm/lxc instances, we can use Drive Client (SeaDrive without GUI, but only for one client)
+ Long support period
+ Two versions to choose from: CE-->Community Edition PE-->Professional Edition (up to three users for free).
+ Docker and a downloadable package with the whole server (we install with documentation)

- Only WebDAV and FUSE (in addition to link sharing) extensions that we need to configure additionally
- For getting other integrations, too, configuration is necessary

![202402151613](https://github.com/tteck/Proxmox/assets/6137813/416c8e15-e278-4541-8046-c8d01b02a540)
![202402151614](https://github.com/tteck/Proxmox/assets/6137813/55a0e473-dedf-4ffe-a56b-877c28aefd93)

On [github](https://github.com/haiwen/seafile-server-installer) there is "Auto install Seafile Server"

I think that for pure basics the script would be ok (if all the rest is configured individually by everyone).
Otherwise, I suspect that the script will be terribly cumbersome (by the mass of Issues)

Unfortunately, I do not have an formed opinion. I need to personally test Seafile(Pro) and FileCloud in the community version.

**Comment by protonaut** [comment]
+1
